### DISCUSSION TITLE
Forked Goerli, barely any tx work

### DISCUSSION THREAD

**Discussion by pclaesen** [ConsenSys-archive/ganache#4264] [General]
I forked Goerli with Ganache, but barely any of my tx go through.
Contract deployment with Truffle dashboard keep pending, need to deploy it old school.
Using 2.5 eth for a token approval stays pending forever. Is there any way to set super low gas fees on all levels of the fork?
Or could there be another reason?

**Comment by jeffsmale90** [comment]
I wouldn't expect any issues with sending transactions to a forked Goerli network. 

Can you please share more details of the issue - ideally with reproduction steps so that we can debug this.

Thanks!

**Reply by pclaesen** [reply]
I forked Goerli with this command:
`ganache --fork.network goerli;`

I just forked it again during UTC time 9AM with a miner blocktime specified instead of the default:
`ganache --fork.network goerli --miner.blockTime=2; `

Now my tx work. So either the mining bugs out with the default of 0 (instamine), or it has something to do with Goerli fees/network usage being lower at the moment of forking.

**Reply by jeffsmale90** [reply]
That's great!

If there is a problem in Ganache, I'd definitely be keen to fix it, so it'd be great if you could share a reproduction. If you don't have time, that's fine - please just mark the discussion as resolved.

Thanks!

**Reply by pclaesen** [reply]
> That's great!
> 
> If there is a problem in Ganache, I'd definitely be keen to fix it, so it'd be great if you could share a reproduction. If you don't have time, that's fine - please just mark the discussion as resolved.
> 
> Thanks!

Hi Jeff, It's basically just using the normal fork command `ganache --fork.network goerli;` that gives the problem (sorry for my late reply).
